journal/202012.md

```diff
@@ -0,0 +1,334 @@
+- save [client info](https://github.com/decred/dcrpool/pull/293) in the database and make it available on any dcrpool instance
+- support for NiceHash pool [validator](https://github.com/decred/dcrpool/pull/294) (allows it to connect to dcrpool and query basic stats) { is this description correct? }
+
+**[dcrdex](https://github.com/decred/dcrdex)**
+
+- new [max order](https://github.com/decred/dcrdex/pull/842) button that populates quantity fields with the maximum amount possible based on market, market side and wallet balance
+- public [market data](https://github.com/decred/dcrdex/pull/796) API endpoints via HTTP and WebSockets
+- show [account ID](https://github.com/decred/dcrdex/pull/825) for each DEX server
+- favor [confirmed](https://github.com/decred/dcrdex/pull/865) outputs for funding BTC orders
+- [resume](https://github.com/decred/dcrdex/pull/856) swapper from the database (better than the previous approach and less code)
+- allow apps to listen on [IPv6](https://github.com/decred/dcrdex/pull/899) addresses
+- improve colors to differentiate between buy and sell orders
+- many bug fixes and dependency upgrades
+
+A total of 28 PRs from 7 contributors were [merged](https://github.com/decred/dcrdex/pulls?q=is%3Apr+merged%3A2020-12-01..2020-12-31+sort%3Aupdated-asc), adding 8K and deleting 5K lines of code.
+
+Patch release [v0.1.4](https://github.com/decred/dcrdex/milestone/10) is getting close.
+
+**[dcrandroid](https://github.com/planetdecred/dcrandroid)**
+
+- notification when cfilters are [fetched](https://github.com/planetdecred/dcrandroid/pull/523)
+- bug fixes
+
+Merged in dcrlibwallet shared library:
+
+- [vspd](https://github.com/planetdecred/dcrlibwallet/pull/163) support
+- method to get info about connected SPV [peers](https://github.com/planetdecred/dcrlibwallet/pull/160)
+- dependency upgrades and bug fixes
+
+Note that features added to dcrlibwallet become available for dcrandroid, dcrios and godcr (assuming they add UI for it).
+
+**[dcrios](https://github.com/planetdecred/dcrios)**
+
+- [automated](https://github.com/planetdecred/dcrios/pull/707) UI tests for some scenarios using XCUI
+
+**[godcr](https://github.com/planetdecred/godcr)**
+
+- [coin selection](https://github.com/planetdecred/godcr/pull/261) interface
+- [notifications](https://github.com/planetdecred/godcr/pull/274) when text is copied or transaction is created
+- new UI design for main [Wallets](https://github.com/planetdecred/godcr/pull/276) page
+- UI tweaks
+- bug fixes and optimizations
+
+{ any comment on communication/interaction between godcr <> gio projects? }
+
+**[tinydecred](https://github.com/decred/tinydecred)**
+
+In progress:
+
+- script to [recover](https://github.com/decred/tinydecred/pull/194) funds from an old Copay web wallet's xpriv (extended private key). The script has already helped to recover someone's mainnet funds. { is it a correct description? does "web wallet" refer to the one that was at wallet.decred.org? }
```

Comment:
I believe the description is correct. I have no idea where the "web wallet" was being used. Maybe someone else knows?